res/themes/Dark Blue/client/client.css

```
@@ -0,0 +1,1618 @@
+}
+
+QToolButton::hover
+{
+    border: none;
+    color: #E5E7EB;
+    padding: 5px;
+    background-color: #374151;
+    border-radius: 6px;
+}
+
+QToolButton::checked
```

Biome (2.1.2)

[error] Unknown type selector is not allowed.

See MDN web docs for more details.
Consider replacing the unknown type selector with valid one.

(lint/correctness/noUnknownTypeSelector)